react/basic-example/src/components/Chatbox.js

```diff
@@ -0,0 +1,39 @@
+import React, { useContext, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { TalkSessionContext } from '../contexts/TalkSessionContext';
+
+/* 
+This is an example component that uses one of TalkJS' pre-built UI modes - or more specifically, the 'Chatbox' UI mode
+The Chatbox UI is typically used when the chat may be related to a speicific 'topic' such as a video, product order etc.
```

Comment:
```suggestion
The Chatbox UI is typically used when the chat may be related to a specific 'topic' such as a video, product order etc.
```

Reply:
Whoops nice find. Fixed in incoming commit